test/elixir_analyzer/test_suite/dna_encoding_test.exs

```diff
@@ -0,0 +1,226 @@
+defmodule ElixirAnalyzer.ExerciseTest.DNAEncodingTest do
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.DNAEncoding
+
+  test_exercise_analysis "example solution",
+    comments: [ElixirAnalyzer.Constants.solution_same_as_exemplar()] do
+    defmodule DNA do
+      @moduledoc """
```

Comment:
@angelikatyborska note for later, this moduledoc will make it impossible for anyone to get the celebratory comment. Once the representer goes live, we should consider comparing solutions through it. In the mean time, maybe we can remove the moduledoc in the exemplar?

Reply:
Of course! I'm surprised it's even there. I created a PR https://github.com/exercism/elixir/pull/1020